ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -100,7 +100,7 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
 
 RCT_REMAP_METHOD(userProvidedPrivacyConsent, resolver: (RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
-    resolve(@(!OneSignal.requiresUserPrivacyConsent));
+    resolve(@(![OneSignal requiresUserPrivacyConsent]));
```

Comment:
nit, using dots are cleaner if it is just a getter.

Reply:
okay gotcha sorry, changing back